flopy/mbase.py

```diff
@@ -696,19 +696,29 @@ def __getattr__(self, item):
         Parameters
         ----------
         item : str
-            3 character package name (case insensitive) or "sr" to access
-            the SpatialReference instance of the ModflowDis object
+            This can be one of:
+
+                * A 3-character package name (case insensitive) returns package
+                * "tr" to access the time discretization object
+                * "modelgrid" to access the spatial discretization object
```

Comment:
looks like nothing is returned if item is "modelgrid", I'm not sure whether that was original intent, should the grid be returned in that case or the option removed?

Reply:
Actually it turns out that "modelgrid" is not possible, since this is overshadowed by a class property (see them with `git grep -B1 "def modelgrid"`). I'll test a few more of these out and adjust this method and docs a bit better.

Reply:
It seems that there are only a few instances where "modelgrid" or "nper" could be accessed. I've noted these for now. This feature seems to be a bit broken for ModflowLgr too.

Reply:
thanks @mwtoews it seems this will need some work to straighten out.